### ISSUE TITLE
Strikethrough text selection on Orphan annotations reportedly not accessible to screenreader users

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/product-backlog#1634]
From the HighWire Press / McGraw-Hill Accessibility spreadsheet: https://hypothes-is.slack.com/archives/C8TPC8XMK/p1736535764817829

**Highwire Press Reproduction**
1. Inspect the code of the mentioned content.
2. Notice that the content is mentioned through the CSS content property and not the HTML code.

Compare:
1. https://hyp.is/Eo7VcC_fEeq1Lgdf1x2UVw/example.com/
2. https://hyp.is/hTpVxKgcEeyii_dQpNUJlg/via.hypothes.is/ (as viewed on example.com due to document equivalence)

```
<div data-testid="excerpt-container" class="relative overflow-hidden transition-[max-height] ease-in duration-150" style="max-height: 18px;"><div class="inline-block w-full" data-testid="excerpt-content"><div dir="auto" lang="" class="StyledText"><blockquote class="hover:border-l-blue-quote">Example Domain</blockquote></div></div><div data-testid="excerpt-expand" role="presentation" class="transition-[opacity] duration-150 ease-linear absolute w-full bottom-0 h-touch-minimum bg-none opacity-0 pointer-events-none" title="Show the full excerpt"></div></div>

<div data-testid="excerpt-container" class="relative overflow-hidden transition-[max-height] ease-in duration-150" style="max-height: 36px;"><div class="inline-block w-full" data-testid="excerpt-content"><div dir="auto" lang="" class="StyledText p-redacted-text"><blockquote class="hover:border-l-blue-quote">Via is a closed proxy that lets you annotate web pages and PDFs with Hypothesis.</blockquote></div></div><div data-testid="excerpt-expand" role="presentation" class="transition-[opacity] duration-150 ease-linear absolute w-full bottom-0 h-touch-minimum bg-none opacity-0 pointer-events-none" title="Show the full excerpt"></div></div>

```

Note: the only different I can find is the strikethrough text having "p-redacted-text" in the class.

**HighWire Expected Result**
The informative information should be available to all users in all/any platforms.

**HghWire Actual Result**
The screen reader does not announce the strikethrough content in the "Orphans" tab that becomes available on activating the "Annotation sidebar" button as the content is defined through CSS content property.
As a results, screen reader users might not be able to access the content.

Code snippet:
```
<div class=""inline-block w-full"" data-testid=""excerpt-content"">
(...)
<blockquote class=""hover:border-l-blue-quote"">Significant and persistent changes in the state of Earth’s climate</blockquote>
</div>
```

**HighWire Rationale**
When content is not available for screen reader users, the screen reader users cannot access the information conveyed by the content. 

**HighWire Recommendation**
It is recommended to provide the hidden text as ""Strikethrough text starts"" and ""Strikethrough text ends"".

Suggested Code:
HTML:
```
<div class=""inline-block w-full"" data-testid=""excerpt-content"">
<p class=sr.only> Strikethrough content starts</p>
(...)
<blockquote class=""hover:border-l-blue-quote"">Significant and persistent changes in the state of Earth’s climate</blockquote>
<p class=sr.only> Strikethrough content ends</p>
</div>
```

CSS:
```
.sr.only {
position: absolute;
left: -9999px;
}

```
